paytest/test_paytest.py

```diff
@@ -41,9 +41,9 @@ def test_mpp_pay(node_factory):
     """ l1 send a payment that is going to be split.
     """
     l1, l2 = node_factory.line_graph(2, opts=pluginopt, wait_for_announce=True)
-    res = l1.rpc.paytest(l2.info['id'], 10**8)
+    res = l1.rpc.paytest(l2.info['id'], 2*10**8)
```

Comment:
Out of curiosity: Is this test more stable with 200000000 ?

Reply:
Just needs enough value to force a split which this does.